library/ssl_tls13_client.c

```diff
@@ -2390,7 +2392,7 @@ static int ssl_certificate_request_coordinate( mbedtls_ssl_context* ssl )
 #else
     if( ssl->keep_current_message == 0 )
     {
-        if( ( ret = mbedtls_ssl_read_record( ssl ) ) != 0 )
+        if( ( ret = mbedtls_ssl_read_record( ssl, 1 ) ) != 0 )
```

Comment:
Why didn't you want to put the SSL_FORCE_FLUSH here (and below as well)?

Reply:
Hm, good question, maybe because it wasn't defined in this file? I'll check.